hw/dv/sv/flash_phy_prim_agent/doc/index.md

```diff
@@ -1,3 +1,5 @@
-# FLASH_PHY_PRIM UVM Agent
+---
+title: Flash Physical Primitive UVM Agent
```

Comment:
I'd suggest calling it `Phy` because it is a well known industry term.

Reply:
Thanks, I agree.